style.css

```diff
@@ -1,324 +1,473 @@
+        -ms-flex-pack: center;
+        -ms-flex-wrap: wrap;
+            flex-wrap: wrap;
+        justify-content: center;
+    }
+    .voyage {
+        width: 100%;
+        max-width: 345px;
+        height: 508px;
+        padding: 25px 15px 0 15px;
+    }
+    .advert__title {
+        letter-spacing: 1.3px;
+
+        font-size: 26px;
+        line-height: 1.85;
+    }
+    .advert__subtitle {
+        margin-bottom: 23px;
+
+        letter-spacing: -1.8px;
+
+        font-size: 72px;
+        line-height: 0.67;
+    }
+    .voyage__picture {
+        width: 345px;
+        height: 272px;
+    }
+    .voyage__descr {
+        min-width: 296px;
+        height: 207px;
+    }
+    .footer {
+            flex-direction: row;
+
+        align-items: center;
+        -webkit-box-align: center;
+        -webkit-box-direction: normal;
+        -webkit-box-orient: horizontal;
+        -ms-flex-align: center;
+        -ms-flex-direction: row;
+    }
 }
 @media (min-width: 1440px) {
-  .header {
-    height: 648px;
-  }
-  .advert__logo {
-    width: 88px;
-    height: 88px;
-
-    background-image: url("images/header__logo_desktop.png");
-  }
-  .advert__title {
-    letter-spacing: 1.8px;
-
-    font-size: 36px;
-    line-height: 1.33;
-  }
-  .advert__subtitle {
-    margin-bottom: 57px;
-
-    letter-spacing: -2.5px;
-
-    font-size: 100px;
-  }
-  .voyage {
-    width: 100%;
-    max-width: 555px;
-    margin-top: 21px;
-  }
-  .voyage__picture {
-    width: 555px;
-    height: 215px;
-  }
-  .voyage__picture_large {
-    height: 437px;
-  }
-  .voyage__picture_small {
-    height: 113px;
-  }
-  .voyage__picture_small > img {
-    top: -75%;
-  }
-  .voyage_for-desktop {
-    display: block;
-  }
-}
+    .header {
+        height: 648px;
+    }
+    .advert__logo {
+        width: 88px;
+        height: 88px;
+
+        background-image: url("images/header__logo_desktop.png");
+    }
+    .advert__title {
+        letter-spacing: 1.8px;
+
+        font-size: 36px;
+        line-height: 1.33;
+    }
+    .advert__subtitle {
+        margin-bottom: 57px;
+
+        letter-spacing: -2.5px;
+
+        font-size: 100px;
+    }
+    .catalog-list {
+        display: -ms-grid;
+        display:     grid;
+
+        width: 1140px;
+
+        -ms-grid-columns: 555px 30px 555px;
+        grid-template-columns: 555px 30px 555px;
+    }
+    .voyage {
+        -webkit-box-sizing: content-box;
+                box-sizing: content-box;
+        width: 100%;
+        max-width: 555px;
+        height: auto;
+    }
+    .voyage:first-child {
+        -ms-grid-column: 1;
+            grid-column: 1 / 2;
+        -ms-grid-row: 1;
+            grid-row: 1 / 3;
+        -ms-grid-row-span: 2;
+    }
+    .voyage:nth-child(2) {
+        -ms-grid-column: 2;
+        -ms-grid-row: 1;
+    }
+    .voyage:nth-child(3) {
+        -ms-grid-column: 3;
+        -ms-grid-row: 1;
+    }
+    .voyage:nth-child(4) {
+        -ms-grid-column: 2;
+        -ms-grid-row: 2;
```

Comment:
I'm really appreciate that you  found solution for IE :thumbsup:

Unfortunately this will not work in real project with random amount of items
You can do the with help flex